canonical-web-and-design/juju.is
# Add syntax highlighting to docs

## Done
Added syntax highlighting to code examples in docs pages to improve readability

## QA
- Go to:
  - https://juju-is-253.demos.haus/docs/sdk/charm-anatomy
  - https://juju-is-253.demos.haus/docs/sdk/constructs
- Check that the code examples have syntax highlighting

## Issue
Fixes #190
